src/Controller/ModulesController.php

```diff
@@ -82,6 +82,17 @@ public function index(): ?Response
 
         $query = $this->request->getQueryParams();
 
+        // sort by descending id if no other sort is specified
+        if (empty($query['sort'])) {
```

Comment:
this is now obsolete after https://github.com/bedita/manager/pull/422 - you can decide a default `sort` on each module